home_work_11/index.js

```diff
@@ -0,0 +1,108 @@
+import { mockDataArray } from "./mockData.js";
+
+const root = document.createElement("div");
+
+const containerValueOfProducts = document.createElement("div");
+containerValueOfProducts.classList.add("container");
+
+const productList = document.createElement("ul");
+productList.classList.add("product-list");
+
+root.prepend(productList, containerValueOfProducts);
+document.body.prepend(root);
+
+function showDetails(productName) {
+  const productDetailsBlock = document.getElementById(productName);
+  if (productDetailsBlock.style.display === "none") {
+    productDetailsBlock.style.display = "block";
+    return;
```

Comment:
Here you can do so:
`productDetailsBlock.classList.toggle('hidden-product'))`

And add styles to that class in CSS: 

```
.hidden-product {
    display: none
}
```